### ISSUE TITLE
Different Harakiri per app

### ISSUE THREAD

**Issue by dav009** [unbit/uwsgi#1303]
Is it possible to have a different harakiri time per app ?

**Comment by unbit** [comment]
Hi, if you mean multiple apps in the same process you can use the internal routing framework with the 'harakiri' action, so you can set harakiri per-request:

```ini
[uwsgi]
route = ^/foobar harakiri:30
route = ^/test harakiri:60
```

**Comment by dav009** [comment]
awesome 👍

**Comment by dav009** [comment]
since what version is this available? Im currently using `2.0.7` but I cant see any changes in the harakiri behaviour

**Comment by unbit** [comment]
Ensure your instance is compiled with internal routing support (requires libpcre3)
